### ISSUE TITLE
View job postings

### ISSUE THREAD

**Issue by SahanaCoding** [LachezaraLaz/341-soen341project2023#80]
`<!DOCTYPE` html>
<html>

<head>
  <link rel="stylesheet" href="../static/css/jobDashboardCSS.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
  <script src="https://kit.fontawesome.com/e4ad6adbad.js" crossorigin="anonymous"></script>

  
  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet">

  
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
  <style>
    .custom {
      width: 40%;
    }
  </style>

</head>

<body>

  <div class="container-fluid">
    <div class="row">
      <ul class="nav-bar">
        <li><a class="name" href="../Login/Login-html%20file.html"><b><i>I</i>mplico</b></a></li>
        <li><a href="../Profile/profile.html"><i class="fa fa-fw fa-user"></i> Me</a></li>
        <li><a href="#"><i class="fa fa-fw fa-envelope"></i> Jobs</a></li>
        <li><a class="active" href="../Login/Login-html%20file.html" href="#"><i class="fa fa-fw fa-home"></i> Home</a>
        </li>
      </ul>
    </div>
    <br><br><br>

    <div class="row">
        <div class="h-100 d-flex align-items-left justify-content-center">
          <div class="col-8 ps-5 pt-4 pb-4 pe-4 bg-light rounded">
            <div class="row px-3 py-3 text-left">
              
            </div>

            <div class="container d-flex flex-row border border-4">
                
                   
             <div class="centered" style=" align-items:center;width: 60%; display: flex;"><h4>The Right Jobs for You</h4> </div>
            
                <img src="../static/HappyMan.jpg" alt="HappyMan" class="img-fluid" style="width:50%;">

            </div>

            
            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
            <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
            <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
            

          
            
              <div class="container">
                <form class="d-flex ps-3 pt-2 pb-2 pe-2 text-left justify-content-start align-items-center">
                  <div class=" d-flex me-2">
                    <input type="search" class="form-control" placeholder="Search" aria-label="Search">
                    <button class="btn btn-info text-light" type="submit">Search</button>

                  </div>

                  <div class=" d-flex me-2">
                    <input type="text" class="form-control" placeholder="Montreal,Qc">
                    <button class="btn btn-info text-light" type="button">Locate</button>
                  </div>

                  <div class= "d-flex me-2">
                    <input type="text" class="form-control" placeholder="Find Jobs">
                    <button class="btn btn-info text-light" type="button">Find Jobs</button>
                  </div>

                </form>
              </div>
              
    <div class="row">
      <div class="h-100 d-flex align-items-left justify-content-left">
        <div class="col-8 ps-2 pt-2 pb-2 pe-3 w-100 bg-light rounded ">
          <div class="row px-2 py-2 text-left ">
          </div>

                <div class="row px-3 py-3 text-left">
                  <h5>Recommended For You</h5>

                </div>

                <div class="container p-3 my-3 bg-white text-black border border-4">

                  <div class="form-group">
                    <label for="jobPosition">{{Job 1 Position}}</label> <br>
                    <label for="companyName" style= font-size:80%>{{Job 1 company name}}</label> <br>
                    <label for="Location"style= font-size:80%>{{Location (Remote or Hybrid)}} </label> <br>

                  </div>

                  
          </div>

          <div class="container p-3 my-3 bg-white text-black border border-4">

            <div class="form-group">
              <label for="jobPosition">{{Job 2  Position}}</label> <br>
              <label for="companyName" style= font-size:80%>{{Job 2 company name}}</label> <br>
              <label for="Location"style= font-size:80%>{{Location (Remote or Hybrid)}} </label> <br>

            </div>
          

        </div>

        <div class="container p-3 my-3 bg-white text-black border border-4">

          <div class="form-group">
            <label for="jobPosition">{{Job 3  Position}}</label> <br>
            <label for="companyName" style= font-size:80%>{{Job 3 company name}}</label> <br>
            <label for="Location"style= font-size:80%>{{Location(Remote or Hybrid)}} </label> <br>

          </div>

      </div>
      <br><br><br>

      

</body>

</html>`

**Comment by SahanaCoding** [comment]
`<!DOCTYPE html>
<html>

<head>
  <link rel="stylesheet" href="../static/css/jobDashboardCSS.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
  <script src="https://kit.fontawesome.com/e4ad6adbad.js" crossorigin="anonymous"></script>

  
  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet">

  
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
  <style>
    .custom {
      width: 40%;
    }
  </style>



</head>

<body>

  <div class="container-fluid">
    <div class="row">
      <ul class="nav-bar">
        <li><a class="name" href="../Login/Login-html%20file.html"><b><i>I</i>mplico</b></a></li>
        <li><a href="../Profile/profile.html"><i class="fa fa-fw fa-user"></i> Me</a></li>
        <li><a href="#"><i class="fa fa-fw fa-envelope"></i> Jobs</a></li>
        <li><a class="active" href="../Login/Login-html%20file.html" href="#"><i class="fa fa-fw fa-home"></i> Home</a>
        </li>
      </ul>
    </div>
    <br><br><br>


    <div class="row">
        <div class="h-100 d-flex align-items-left justify-content-center">
          <div class="col-8 ps-5 pt-4 pb-4 pe-4 bg-light rounded">
            <div class="row px-3 py-3 text-left">
              
            </div>

            <div class="container d-flex flex-row border border-4">
                
    
                   
             <div class="centered row" style=" align-items:center;width: 60%; display: flex;"><h4>The Right Jobs for You</h4> </div>
            
                <img src="../static/HappyMan.jpg" alt="HappyMan" class="img-fluid" style="width:50%;">

            </div>

            
            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
            <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
            <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
            

          
            
              
             
    <div class="row">
      <div class="h-100 d-flex align-items-left justify-content-left">
        <div class="col-8 ps-2 pt-2 pb-2 pe-3 w-100 bg-light rounded ">
          <div class="row px-2 py-2 text-left ">
          </div>

                <div class="row px-3 py-3 text-left">
                  <h5>Recommended For You</h5>

                </div>

                <div class="container p-3 my-3 bg-white text-black border border-4">

                  <div class="form-group">
                    <label for="jobPosition">{{Job 1 Position}}</label> <br>
                    <label for="companyName" style= font-size:80%>{{Job 1 company name}}</label> <br>
                    <label for="Location"style= font-size:80%>{{Location (Remote or Hybrid)}} </label> <br>

                  </div>

                  
          </div>

          <div class="container p-3 my-3 bg-white text-black border border-4">

            <div class="form-group">
              <label for="jobPosition">{{Job 2  Position}}</label> <br>
              <label for="companyName" style= font-size:80%>{{Job 2 company name}}</label> <br>
              <label for="Location"style= font-size:80%>{{Location (Remote or Hybrid)}} </label> <br>

            </div>
          

        </div>

        <div class="container p-3 my-3 bg-white text-black border border-4">

          <div class="form-group">
            <label for="jobPosition">{{Job 3  Position}}</label> <br>
            <label for="companyName" style= font-size:80%>{{Job 3 company name}}</label> <br>
            <label for="Location"style= font-size:80%>{{Location(Remote or Hybrid)}} </label> <br>

          </div>

      </div>
      <br><br><br>

      <div class="text-center">
      <button type="edit" class="btn btn-primary custom ">Edit</button>
      <button type="submit" class="btn btn-primary custom ">Add</button> 
      </div>
</body>

</html>`

**Comment by SahanaCoding** [comment]
edit and add buttons were added
